operators/test/e2e/test/command/kubectl.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package command
+
+import (
+	"fmt"
+)
+
+// Kubectl represents the kubectl command.
+type Kubectl struct {
+	defaultArgs []string
+}
+
+func NewKubectl(kubeconfigPath string) *Kubectl {
+	k := &Kubectl{}
+	if kubeconfigPath != "" {
+		k.defaultArgs = []string{fmt.Sprintf("--kubeconfig=%s", kubeconfigPath)}
+	}
+
+	return k
+}
+
+func (k *Kubectl) Command(command string, args ...string) *Command {
+	argList := make([]string, len(k.defaultArgs)+len(args)+1)
+	argList[0] = command
+	copy(argList[1:], k.defaultArgs)
+	copy(argList[len(k.defaultArgs)+1:], args)
+
+	return New("kubectl", argList...).Build()
+}
+
+// Apply executes "kubectl apply"
+func (k *Kubectl) Apply(args ...string) *Command {
```

Comment:
Does not seem to be used, `kubectlApplyTemplate` seems to use `Command` directly  (https://github.com/elastic/cloud-on-k8s/pull/1364/files#diff-0f1797d6d16a11ac4a0f264a07f3ce54R366),